tests/component/conftest.py

```diff
@@ -0,0 +1,46 @@
+import os
```

Comment:
why are you using python and not go as we have for unit tests? I ask because perhaps we could either reuse the unit test framework or at least part of it for component tests. Also to avoid a third language, when already have 2 (go and bash) for tests.